packages/devtools_test/lib/utils.dart

```diff
@@ -297,3 +294,23 @@ void verifyIsSearchMatch(
     }
   }
 }
+
+Future<void> waitFor(
```

Comment:
this looks like it needs to be formatted with dartfmt

Reply:
Thanks, just updated this after running dartfmt from packages/ directory.